src/test/java/com/wikia/webdriver/common/core/helpers/Emulator.java

```diff
@@ -1,9 +1,84 @@
 package com.wikia.webdriver.common.core.helpers;
 
-public class Emulator {
+import com.google.common.collect.ImmutableMap;
+import lombok.Getter;
 
-  public static final String GOOGLE_NEXUS_5 = "Google Nexus 5";
+import java.util.Map;
 
-  private Emulator() {
+public enum Emulator {
+  GOOGLE_NEXUS_5(
+      new ImmutableMap.Builder<String, Object>()
+          .put("width", 360)
+          .put("height", 640)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "height" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)